frontend/src/httpClient.ts

```diff
@@ -57,10 +57,15 @@ const updateProfile = async (request: UpdateProfileRequest) => {
   return response.data;
 };
 
+const deleteProfile = async () => {
+  const response = await httpClient.delete("delete-profile");
```

Comment:
I guess we can refactor things a bit. `delete-profile` might be not the best name for an endpoint, since it is already a delete method. Having `delete` in the text might be a burden later.

Reply:
We can can keep is as is for now, we will revisit our endpoint naming/conventions later.